pkg/webhook/pod/validating/cluster_reservation.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package validating
```

Comment:
The name of this file is very strange. What does it have to do with reservation? It's OK to just call verify_annotations.go.

Reply:
You're a professional.👍🏻